main.go

```diff
@@ -393,8 +411,18 @@ func ParsePipelineRunList() *cobra.Command {
 					fmt.Fprintf(w, str)
 				}
 			}
+			switch outputType {
+			case OutputTypeCsv:
+				fmt.Fprintln(w, "PipelineRun;Duration;Concurrency")
+			default:
+			}
 			for i, key := range retS {
-				fmt.Fprintf(w, "PipelineRun %s\t\ttook %v seconds with concurrency at %d\n", key, retF[i], retI[i])
+				switch outputType {
```

Comment:
ok now that we are more than just a single line `fmt.Fprintf` call let's go ahead and create a utility function for this common pattern 

your new fmt.Fprintf for the csv looks common throughout

for the human readable string, pass in the format string as a parameter, along with a `string...` parameter, i.e. a "variadic function" parmeter  for the variable length on params to fill into that format string

refs:  https://gobyexample.com/variadic-functions and https://stackoverflow.com/questions/27169110/how-to-pass-variable-length-arguments-as-arguments-on-another-function-in-golang